### ISSUE TITLE
Couldn't find match for /refs 

### ISSUE THREAD

**Issue by nids2307** [cypress-io/cypress#6895]
### Current behavior:

Unable to upgrade cypress or install yarn locally on the project

I see the following error

```
 PS C:\Users\ns_project> yarn add cypress@4.2.0
yarn add v1.19.1
[1/4] Resolving packages...
error Couldn't find match for "b5af0d1fa47eec97ba980cde90a13e69a2afcd16" in "refs/heads/3.0,refs/heads/fix-firefox-infinite-recursion,refs/heads/greenkeeper-aws4-1.6.0,refs/heads/greenkeeper-browserify-14.0.0,refs/heads/greenkeeper-browserify-14.2.0,refs/heads/greenkeeper-browserify-14.3.0,refs/heads/greenkeeper-browserify-14.4.0,refs/heads/greenkeeper-mime-types-2.1.12,refs/heads/greenkeeper-standard-10.0.0,refs/heads/greenkeeper-standard-10.0.1,refs/heads/greenkeeper-standard-10.0.2,refs/heads/greenkeeper/remove-node-0.10,refs/heads/master,refs/heads/more-precise-test,refs/heads/revert-2808-pgonzal/issue2807,refs/heads/standard-style,refs/heads/tap,refs/pull/1/head,refs/pull/2/head,refs/tags/v1.2.0,refs/tags/v2.17.0,refs/tags/v2.18.0,refs/tags/v2.18.1,refs/tags/v2.19.0,refs/tags/v2.19.1,refs/tags/v2.20.0,refs/tags/v2.20.1,refs/tags/v2.21.0,refs/tags/v2.21.1,refs/tags/v2.22.0,refs/tags/v2.22.1,refs/tags/v2.23.0,refs/tags/v2.23.1,refs/tags/v2.24.0,refs/tags/v2.24.1,refs/tags/v2.25.0,refs/tags/v2.25.1,refs/tags/v2.26.0,refs/tags/v2.26.1,refs/tags/v2.27.0,refs/tags/v2.27.1,refs/tags/v2.28.0,refs/tags/v2.28.1,refs/tags/v2.29.0,refs/tags/v2.29.1,refs/tags/v2.30.0,refs/tags/v2.30.1,refs/tags/v2.31.0,refs/tags/v2.31.1,refs/tags/v2.32.0,refs/tags/v2.32.1,refs/tags/v2.33.0,refs/tags/v2.33.1,refs/tags/v2.34.0,refs/tags/v2.34.1,refs/tags/v2.35.0,refs/tags/v2.35.1,refs/tags/v2.36.0,refs/tags/v2.36.1,refs/tags/v2.37.0,refs/tags/v2.37.1,refs/tags/v2.38.0,refs/tags/v2.38.1,refs/tags/v2.39.0,refs/tags/v2.39.1,refs/tags/v2.40.0,refs/tags/v2.40.1,refs/tags/v2.41.0,refs/tags/v2.41.1,refs/tags/v2.42.0,refs/tags/v2.42.1,refs/tags/v2.43.0,refs/tags/v2.43.1,refs/tags/v2.44.0,refs/tags/v2.44.1,refs/tags/v2.45.0,refs/tags/v2.45.1,refs/tags/v2.46.0,refs/tags/v2.46.1,refs/tags/v2.47.0,refs/tags/v2.47.1,refs/tags/v2.48.0,refs/tags/v2.48.1,refs/tags/v2.49.0,refs/tags/v2.49.1,refs/tags/v2.50.0,refs/tags/v2.50.1,refs/tags/v2.51.0,refs/tags/v2.51.1,refs/tags/v2.52.0,refs/tags/v2.52.1,refs/tags/v2.53.0,refs/tags/v2.53.1,refs/tags/v2.54.0,refs/tags/v2.54.1,refs/tags/v2.55.0,refs/tags/v2.55.1,refs/tags/v2.56.0,refs/tags/v2.56.1,refs/tags/v2.57.0,refs/tags/v2.57.1,refs/tags/v2.58.0,refs/tags/v2.58.1,refs/tags/v2.59.0,refs/tags/v2.59.1,refs/tags/v2.60.0,refs/tags/v2.60.1,refs/tags/v2.61.0,refs/tags/v2.61.1,refs/tags/v2.62.0,refs/tags/v2.62.1,refs/tags/v2.63.0,refs/tags/v2.63.1,refs/tags/v2.64.0,refs/tags/v2.64.1,refs/tags/v2.65.0,refs/tags/v2.65.1,refs/tags/v2.66.0,refs/tags/v2.66.1,refs/tags/v2.67.0,refs/tags/v2.67.1,refs/tags/v2.68.0,refs/tags/v2.68.1,refs/tags/v2.69.0,refs/tags/v2.69.1,refs/tags/v2.70.0,refs/tags/v2.70.1,refs/tags/v2.71.0,refs/tags/v2.71.1,refs/tags/v2.72.0,refs/tags/v2.72.1,refs/tags/v2.73.0,refs/tags/v2.73.1,refs/tags/v2.74.0,refs/tags/v2.74.1,refs/tags/v2.75.0,refs/tags/v2.75.1,refs/tags/v2.76.0,refs/tags/v2.76.1,refs/tags/v2.77.0,refs/tags/v2.77.1,refs/tags/v2.78.0,refs/tags/v2.78.1,refs/tags/v2.79.0,refs/tags/v2.79.1,refs/tags/v2.80.0,refs/tags/v2.80.1,refs/tags/v2.81.0,refs/tags/v2.81.1,refs/tags/v2.82.0,refs/tags/v2.82.1,refs/tags/v2.83.0,refs/tags/v2.83.1,refs/tags/v2.84.0,refs/tags/v2.84.1,refs/tags/v2.85.0,refs/tags/v2.85.1,refs/tags/v2.86.0,refs/tags/v2.86.1,refs/tags/v2.87.0,refs/tags/v2.87.1,refs/tags/v2.88.0,refs/tags/v2.88.1,refs/tags/v2.88.2,refs/tags/v2.88.3,refs/tags/v2.88.4,refs/tags/v2.88.5" for "https://github.com/cypress-io/request.git".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

### Desired behavior:

I want to install cypress. I had issues upgrading cypress, so I removed the project and cloned a fresh one from bb repo. I am unable to install yarn as well as cypress due to the above error

### Versions

node.js - v13.11.0
yarn version v1.22.4
OS: Windows 10
git version 2.20.1

**Comment by nids2307** [comment]
I checked the related issues where different users have suggested to clear the yarn cache or update the git version.

I did the same but the issue still persist.

**Comment by jennifer-shehane** [comment]
Relevant bit of error is here:

```
 for "https://github.com/cypress-io/request.git".
```

This was a issue with install in 4.2.0 and has been resolved in 4.3.0. Please `yarn add cypress@4.3.0` 

Duplicate of #6752
